DeMark/src/main/java/actions/ClearAllAction.java

```diff
@@ -22,19 +22,24 @@
     private Project project;        // the IntelliJ project
     private Document document;      // the IntelliJ document
 
-    // TODO: Check the ones that may be null
     // Initializes all fields
     private void init(@NotNull AnActionEvent anActionEvent) {
         editor = anActionEvent.getData(LangDataKeys.EDITOR);
+        if (editor == null) {
+            return;
+        }
         project = editor.getProject();
         document = editor.getDocument();
     }
 
     @Override
     public void actionPerformed(AnActionEvent anActionEvent) {
         init(anActionEvent);
```

Comment:
You're doing two checks to `editor == null` here. Once in `init` and the other after the `init` call. Is that necessary?